### PR TITLE
Notice of Redemption - Landing, Review, Panels

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#17183

*Description of changes:*
- Notice of Redemption Unit Note implementation
- See ticket for details

Screenshot 2023-09-11 at 4 38 32 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1527-crzn4vtt.web.app

File: ppr-ui/src/components/unitNotes/UnitNotePanel.vue [hunk 1]
```diff
@@ -132,13 +132,24 @@ export default defineComponent({
         case UnitNoteDocTypes.NOTE_CANCELLATION:
           cancelUnitNote(note)
           break
+        case UnitNoteDocTypes.NOTICE_OF_REDEMPTION:
+          setMhrUnitNote(prefillUnitNote(note, option))
+          router.push({ path: '/' + RouteNames.MHR_INFORMATION_NOTE })
```
Comment (cameron-eyds) [inline ppr-ui/src/components/unitNotes/UnitNotePanel.vue:137]: You can push to named route without concatenating. 
Or `goToRoute()` from `useNavigation` etc.

File: ppr-ui/src/store/store.ts [hunk 2]
```diff
@@ -626,7 +626,9 @@ export const useStore = defineStore('assetsStore', () => {
     // Current state is to verify the property exists. Future state may be more granular dependent on type.
     return !!state.value.mhrInformation.lienRegistrationType
   })
-  const getMhrUnitNotes: ComputedRef<Array<UnitNoteIF>> = computed<Array<UnitNoteIF>>(() => {
+  const getMhrUnitNotes: ComputedRef<Array<UnitNoteIF | CancelUnitNoteIF>> = computed<
+    Array<UnitNoteIF | CancelUnitNoteIF>
+  >(() => {
     return state.value.mhrUnitNotes
   })
```
Comment (cameron-eyds) [inline ppr-ui/src/store/store.ts]: formatting is a little interesting here, hard to decipher the computed type from broken up like that.

**Review by cameron-eyds — APPROVED** [review]
Looks great, just a few small comments!

File: ppr-ui/src/store/store.ts (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/store/store.ts]: Haha yea didn't even notice, I stopped using a formatter a long time ago, and only use to format long lines, I'll try a different formatter.

**Review by doug-lovett — APPROVED** [review]
Looks great.

File: ppr-ui/src/components/unitNotes/UnitNotePanel.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/unitNotes/UnitNotePanel.vue:137]: I tried `useNavigation`, but it caused strange errors in the tests, which I wasn't able to fix quickly.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1527-crzn4vtt.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1527-crzn4vtt.web.app

**Pull request merged**